src/ui/components/VPNServerCountry.qml

```diff
@@ -36,7 +36,7 @@ VPNClickableRow {
 
     activeFocusOnTab: true
 
-    accessibleName: VPNLocalizer.translateServerCountry(code, name)
+    accessibleName: localizedName
```

Comment:
nit: Maybe call it `localizedCountryName` to make it more obvious what it is

Reply:
This string comes from the `ServerCountryModel` and it matches the `name` attribute.
I like the idea of renaming it, but then I would rename each `name` attribute of each model.
I'll do it in a separate PR.